torch/nn/modules/pooling.py

```diff
@@ -741,6 +741,8 @@ class FractionalMaxPool2d(Module):
     step size determined by the target output size.
     The number of output features is equal to the number of input planes.
 
+    .. note:: Exactly one of `output_size` or `output_ratio` must be defined.
```

Comment:
Sorry one last fix for rendering

```suggestion
    .. note:: Exactly one of ``output_size`` or ``output_ratio`` must be defined.
```